### DISCUSSION TITLE
How to customize created definition

### DISCUSSION THREAD

**Discussion by jjeziorski** [microsoft/vscode-cpptools#11057] [Q&A]
Hello,

I'm using Create Declaration / Definition command, which works great. However, I often need to add some decorations to newly added methods e.g. tracing macros etc depending on project.
Is there a way to customize code (maybe using code snippets) when generating function definition?
If so, how could I set it up?

**Comment by sean-mcmanus** [comment]
Not currently. Can you file a feature request and provide enough information for us to understand what sort of customization you want?
